be/src/olap/page_cache.cpp

```diff
@@ -51,6 +52,10 @@ StoragePageCache::StoragePageCache(size_t capacity, int32_t index_cache_percenta
     } else {
         CHECK(false) << "invalid index page cache percentage";
     }
+    if (pk_index_cache_capacity > 0) {
+        _pk_index_page_cache = std::unique_ptr<Cache>(new_lru_cache(
```

Comment:
Why not let pk_index_page split a part in page cache, like index cache, now the total cache is too large

Reply:
We want pk index page to be seperated from storage page cache cause storage page cache may not enable by default, but pk index cache may always be enabled.